### PR TITLE
fixup! platform: nordic nrf: fix review comments and converting tabs …

### PR DESCRIPTION
**Pull request by ioannisg**
…to spaces

Adding brief descriptions in target_cfg.h headers to clarify
the SPU terminology.

Convert tabs to spaces in source files as requested.

Signed-off-by: Ioannis Glaropoulos <Ioannis.Glaropoulos@nordicsemi.no>

### CODE REVIEW

**Comment by ioannisg** [comment]
@oyvindronningstad FYI

File: platform/ext/target/nordic_nrf/nrf5340/target_cfg.h [hunk 1]
```diff
@@ -18,6 +18,20 @@
...
 
+/**
+ * \file target_cfg.h
+ * \brief nRF5340 target configuration header
+ *
+ * This file contains the platform specific functions to configure
+ * the Cortex-M33 core, memory permissions and security attribution
+ * on the nRF5340 platform.
+ *
+ * Memory permisssions and security attribution are configured via
```
Comment (oyvindronningstad) [inline platform/ext/target/nordic_nrf/nrf5340/target_cfg.h]: We could have a short description in spu.h as well saying that it is our IDAU.

**Review by oyvindronningstad — APPROVED** [review]
LGTM

File: platform/ext/target/nordic_nrf/nrf5340/target_cfg.h (hunk 1 above)
Comment (oyvindronningstad) [reply platform/ext/target/nordic_nrf/nrf5340/target_cfg.h]: typo
```suggestion
 * Memory permissions and security attribution are configured via
```

File: platform/ext/target/nordic_nrf/nrf9160/target_cfg.h [hunk 2]
```diff
@@ -18,6 +18,19 @@
...
 
+/**
+ * \file target_cfg.h
+ * \brief nRF9160 target configuration header
+ *
+ * This file contains the platform specific functions to configure
+ * the Cortex-M33 core, memory permissions and security attribution
+ * on the nRF9160 platform.
+ *
+ * Memory permisssions and security attribution are configured via
```
Comment (oyvindronningstad) [inline platform/ext/target/nordic_nrf/nrf9160/target_cfg.h]:
```suggestion
 * Memory permissions and security attribution are configured via
```

**Comment by oyvindronningstad** [comment]
cc @anangl

**Pull request merged**